catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityFilterPlugin.java

```diff
@@ -44,6 +44,12 @@
     }
 
     public void setAttributeMap(List<String> attributeMappings) {
+        if (attributeMappings == null || attributeMappings.size() == 0 || (
```

Comment:
These first two tests can be replaced with Apache's `CollectionUtils.isEmpty(attributeMappings)`.